extensions/http-curl/client/HTTPClient.h

```diff
@@ -149,6 +149,8 @@ class HTTPClient : public utils::BaseHTTPClient, public core::Connectable {
 
   void setFollowRedirects(bool follow);
 
+  void setMaximumUploadBandwidth(uint64_t max_bandwidth);
+  void setMaximumDownloadBandwidth(uint64_t max_bandwidth);
```

Comment:
This needs a unit throughout the interface. A suffix in the identifiers is good enough, with a comment somewhere. Is it Bps?